index.html

```diff
@@ -1,16 +1,121 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
-</head>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lora&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body>
+    <section class="intro">
+      <div class="intro-img">
+        <img
+          class="intro-img-src"
+          src="./images/cupcake-removebg-preview.png"
+          alt="chocolate cupcake"
+        />
+      </div>
+      <div class="intro-text">
+        <p>The best cakes in town delivered to your door</p>
+      </div>
+    </section>
+    <div class="body">
+      <header class="header">
+        <nav class="navbar">
```

Comment:
Consider how you've built the navbar here -- it's a lot of nesting, making it harder to debug and style. 
something like the below 
```
            <nav class="navbar">

                <a href="#" class="nav-link">Home</a>
                <a href="#" class="nav-link">Cakes</a>
                <a href="#" class="nav-link">Orderin</a>
                <a href="#" class="nav-link">Lessons</a>
                <a href="#" class="nav-link">About</a>

                <div class="hamburger">
                    <span class="bar"></span>
                    <span class="bar"></span>
                    <span class="bar"></span>
                </div>
            </nav>
```